"""
Подвиг 6. В программе объявлена функция для вычисления частного двух чисел:

def get_div(x, y):
    try:
        res = x / y
        return res
    except ZeroDivisionError:
        res = 100
        return res
    finally:
        res = -1
        print(f"finally: {res}")
Выберите все верные утверждения, связанные с этой программой.


при вызове get_div(1, 2) функция возвратит значение -1 (так как блок finally выполняется до оператора return) и на экран будет выведена строчка "finally: -1"
    при вызове get_div(10, 0) функция возвратит значение 100 и на экран будет выведена строчка "finally: -1"
при вызове get_div(10, 0) функция возвратит значение -1 (так как блок finally выполняется до оператора return) и на экран будет выведена строчка "finally: -1"
    при вызове get_div(1, 2) функция возвратит значение 0.5 и на экран будет выведена строчка "finally: -1"
"""
